app/app.go

```diff
@@ -133,12 +143,12 @@ var (
 		stakingtypes.NotBondedPoolName: {authtypes.Burner, authtypes.Staking},
 		govtypes.ModuleName:            {authtypes.Burner},
 		ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
-		oracletypes.ModuleName:         nil,
+		oracletypes.ModuleName:         nil, // TODO:??
 	}
 
 	// module accounts that are allowed to receive tokens
 	allowedReceivingModAcc = map[string]bool{
-		oracletypes.ModuleName: true,
+		oracletypes.ModuleName: true, // TODO: why?
```

Comment:
So, re-reviewing this, could be good to allow transfers to the module account address in the case where we have a set amount of funds for paying out the oracle feeders. But this is pending the TBD economic model here.